main.tscn

```diff
@@ -10,6 +10,7 @@
 script = ExtResource("1_6j1u5")
 
 [node name="ColorRect" type="ColorRect" parent="."]
+z_index = -2
 offset_right = 1280.0
 offset_bottom = 720.0
```

Comment:
@avelitj12 note that if/when you swap out the background, you'll probably need to set `z-index=-2`, since this node will be deleted